tests/unit/misc_test.c

```diff
@@ -183,13 +190,102 @@ CTEST2(misc, test_ctest_assert_prints_user_msg_with_params)
    platform_close_log_stream(&stream, data->log_output);
 }
 
+/*
+ * Simple test to verify lower-level conversion macros used by size_to_str()
+ */
+CTEST2(misc, test_bytes_to_fractional_value_macros)
```

Comment:
Just a brain-dead, `make-sure-I-am-not-making-any-silly-mistakes-test`-test case.